modules/win-autoscale/watchmaker-win-autoscale.template.cfn.yaml

```diff
@@ -475,11 +479,18 @@ Resources:
                 -
                   local_AssignInstanceRole: !If [AssignInstanceRole, !Sub '--role ${InstanceRole}', '']
                   local_UseCfnUrl: !If [UseCfnUrl, !Sub '--url ${CfnEndpointUrl}', '']
-            
               ignoreErrors: 'true'
               waitAfterCompletion: '0'
         install-cloudwatch-agent:
           commands:
+            01-manipulate-cloudwatch-config:
+              command: !Sub
+                - >-
+                  ${local_ShellCommandMap} -Command "
+                  c:\cfn\scripts\update-path.ps1 -Verbose -ErrorAction Stop;
+                  & python c:\cfn\scripts\AmazonCloudWatchAgent\cloudwatch-applog-config.py"
```

Comment:
I think we're close. The last bit is testing each powershell command/script invocation to ensure that any failed command truly results in a non-zero exit. See this bit of code throughout the windows templates... `if (-not $?) { throw 'watchmaker execution failed!' }`

Reply:
added